src/main/resources/Steps/Login.steps

```diff
@@ -0,0 +1,8 @@
+Composite: When I login with email `${email}` and password `${password}`
+When I click on element located by `caseSensitiveText(Log in)`
```

Comment:
I think you could use given i am on the main application page step within precondition as well

Reply:
Commen is still relevent